config/drupal-8/drupal-8.2-deprecations.php

```diff
@@ -2,9 +2,16 @@
 
 declare(strict_types=1);
 
-use DrupalRector\Rector\Deprecation\FileDirectoryOsTempRector;
+use DrupalRector\Rector\Deprecation\FunctionToStaticRector;
 use Rector\Config\RectorConfig;
 
 return static function (RectorConfig $rectorConfig): void {
-    $rectorConfig->rule(FileDirectoryOsTempRector::class);
+    // https://www.drupal.org/node/2802569
+    $rectorConfig->ruleWithConfiguration(FunctionToStaticRector::class, [
+        new \DrupalRector\Rector\ValueObject\FunctionToStaticConfiguration(
+            'file_directory_os_temp',
+            'Drupal\Component\FileSystem\FileSystem',
+            'getOsTemporaryDirectory'
+        ),
+    ]);
```

Comment:
This is from the other PR, right?

Reply:
https://github.com/palantirnet/drupal-rector/pull/251

Reply:
Yeah same code, wanted that as the base for this one, since we going to need that a lot and its pretty simple.

Still, we might merge the other one before this anyways. Here we still need to decide how to handle the fact you migght run this on d9. Then the whole 'copy rector.php' is either invalid, or we need to do magic. Think i might prefer as little magic as possible ;p

Reply:
Also for testing we might need specific rector.php's i think, since the new phpunit set adds a comment to the current drupal 9 set.

Reply:
#251 has been merged.